config/forms/2021_2022.json

```diff
@@ -4365,10 +4365,10 @@
                       "value": "Internal transfer"
                     },
                     "2": {
-                      "value": "Tenant applied directly (no referral or nomination)"
+                      "value": "Tenant applied directly (no referral)"
                     },
                     "3": {
-                      "value": "Private registered provider (PRP) supported lettings only – referred by local authority housing department"
+                      "value": "Referred by local authority housing department"
```

Comment:
@roobottom Do we need to include the text ‘Private registered provider (PRP) supported lettings only’ given the option is only shown in this case anyway?